asset-transfer-basic/rest-api-typescript/src/redis.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * TBC
```

Comment:
TBC...

Reply:
Should probably write something more useful there!!

Reply:
https://github.com/hyperledgendary/fabric-rest-sample/pull/55/files#diff-19955bdd936aa9c7f85937db6ec2aede745e91e8b4de72bb2f6f09a0ea173149R4